### DISCUSSION TITLE
Observing night, August 9, 2023:  Nice results with the eFinder!

### DISCUSSION THREAD

**Discussion by WimDeMeester** [WimDeMeester/eFinder#62] [Show and tell (success stories and failures)]
Yesterday was a clear night (finally) in Belgium.  As I don't have the USB to TTL cable yet, I used version 18 of the eFinder, with the Nexus DSC Pro connected to the eFinder using WiFi, so there was no SkySafari connection. This was the configuration:
![NexusDscPro](https://github.com/WimDeMeester/eFinder/assets/7261529/0ae19a5a-d849-4b67-a1ed-c7b844de17ad)

I could observe for two hours, but I mainly tested the eFinder.  I think I can say that it was a major success!  I first used the VNC GUI version.  There was a little problem there, as on the hand box, I could only see 'Solve Failed'.  The hand box was also not reacting to any action...  Any idea what this could have been, @AstroKeith?  
After about one hour, I switched to using the non GUI version and everything seemed to work fine there.   

As always, I still have a few questions:
- Is the polaris offset saved into the eFinder.config file?  I could not seem to find it...
- Is there a reason why the offset is set to 0.0 after restarting the GUI version?  I could easily reload the offset values, but I don't think that this should be done every time the GUI is restarted...
- How does the non GUI version use the offset?  Is it taken the values from the GUI version automatically?  Or should I do something to measure and use the offset in the non GUI version?

But as I said, the eFinder worked very well and I could look at 20 different objects which were always in the field of view of the telescope after doing a GoTo++.

**Comment by AstroKeith** [comment]
Well some good progress at least!

I’ve uploaded GUI18_1 to the share. Just to make sure you have the latest I have.

If running the GUI, the handpad should show some basics, and allow three actions locally, solve, align, goto. Maybe this is new in 18_1 😙

The offset is stored in the efinder.config as d_x,d_y
The same values are used by handpad and GUI. Just make sure you close the GUI using the ‘x’ at top right of window.
Using the saved value on start up is implemented on GUI18_1 (almost the last line)

**Reply by WimDeMeester** [reply]
I think that the d_x and d_y were still at 0.0, even after closing.  I'll try to do some tests later today to see if I closed the GUI incorrectly...

**Reply by WimDeMeester** [reply]
I was anyway using an older version of the VNC GUI.

**Comment by AstroKeith** [comment]
The GUI16_3 does save what ever the latest saved offset is, on closing ‘normally'
Ver GUI18_1 improves the handpad display handling (a left or right button shows the ‘key’ to commands available)
GUI18_1 also loads the saved offset
GUI18_1 changes the options for image display. Manual/auto rotate is simplified. Plus option to enter a camera frame. Add line ‘frame:30,20’ to efinder.config to define a sensor fov of 30x20 arc min.
GUI18_1 includes ability to enter a goto target at the GUI and upload it to the Nexus DSC

**Reply by WimDeMeester** [reply]
Thanks!  I will try the new version (maybe this evening, as it might be clear.   After that, the secondary mirror really needs to go to the coating company...).
